packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -352,7 +435,7 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget> {
                     ? sum(childrenAndMainAxisSpacesRenderProps
                         .map((renderSize) => renderSize.height))
                     : maxHeight,
-              ),
+              ).normalize(),
```

Comment:
nice!

Reply:
I was running into an issue where resizing the screen would sometimes break the box constraints here.  I suspect it was a precision glitch in the constraint calculations, and normalizing has fixed it.

Reply:
Yes the cross axis empty space calculation is still buggy now, I'll fix that soon, thanks Dave :-)